tests/components/konnected/test_config_flow.py

```diff
@@ -0,0 +1,1052 @@
+                ],
+                "switches": [
+                    {"zone": "4"},
+                    {
+                        "zone": 8,
+                        "name": "switcher",
+                        "activation": "low",
+                        "momentary": 50,
+                        "pause": 100,
+                        "repeat": 4,
+                    },
+                    {"zone": "out1"},
+                    {"zone": "alarm1"},
+                ],
+            }
+        ),
+    )
+    assert result["type"] == "form"
+    assert result["step_id"] == "confirm"
+
+    result = await hass.config_entries.flow.async_configure(
+        result["flow_id"], user_input={}
+    )
+    assert result["type"] == "create_entry"
+    assert result["data"] == {
+        "host": "1.2.3.4",
+        "port": 1234,
+        "id": "112233445566",
+        "model": "Konnected Pro",
+        "access_token": result["data"]["access_token"],
+        "default_options": {
+            "io": {
+                "1": "Disabled",
+                "5": "Disabled",
+                "9": "Disabled",
+                "12": "Disabled",
+                "out": "Disabled",
+                "alarm2_out2": "Disabled",
+                "2": "Binary Sensor",
+                "6": "Binary Sensor",
+                "10": "Binary Sensor",
+                "3": "Digital Sensor",
+                "7": "Digital Sensor",
+                "11": "Digital Sensor",
+                "4": "Switchable Output",
+                "8": "Switchable Output",
+                "out1": "Switchable Output",
+                "alarm1": "Switchable Output",
+            },
+            "blink": True,
+            "discovery": True,
+            "binary_sensors": [
+                {"zone": "2", "type": "door", "inverse": False},
+                {"zone": "6", "type": "window", "name": "winder", "inverse": True},
+                {"zone": "10", "type": "door", "inverse": False},
+            ],
+            "sensors": [
+                {"zone": "3", "type": "dht", "poll_interval": 3},
+                {"zone": "7", "type": "ds18b20", "name": "temper", "poll_interval": 3},
+                {"zone": "11", "type": "dht", "poll_interval": 3},
+            ],
+            "switches": [
+                {"activation": "high", "zone": "4"},
+                {
+                    "zone": "8",
+                    "name": "switcher",
+                    "activation": "low",
+                    "momentary": 50,
+                    "pause": 100,
+                    "repeat": 4,
+                },
+                {"activation": "high", "zone": "out1"},
+                {"activation": "high", "zone": "alarm1"},
+            ],
+        },
+    }
+
+
+async def test_import_existing_config_entry(hass, mock_panel):
+    """Test importing a host that has an existing config entry."""
+    MockConfigEntry(
+        domain="konnected",
+        data={
+            "host": "0.0.0.0",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Possible binding to all interfaces.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4986898)